lib/js/jscore.nim

```diff
@@ -111,14 +111,18 @@ since (1, 5):
   func debugger*() {.importjs: "debugger@".}
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
 
-  template jsExport*(symbol: untyped; alias = "") =
+  template jsExport*(symbol: untyped; name = "") =
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
-    runnableExamples("-b:js -r:off"):
-      proc example = echo "This is exported as 'default' in JavaScript"
-      let example2 = "This is exported as 'another' in JavaScript"
-      const example3 = "This is exported as 'example3' in JavaScript"
-      jsexport(example, alias = "default")  ## Alias for `export { symbol as default };`
-      jsexport(example2, alias = "another") ## Alias for `export { symbol as another };`
-      jsexport(example3)                    ## Alias for `export { symbol };`
+    ##
+    ## .. Note:: Using `name = "default"` `is special in JavaScript <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#using_the_default_export>`_
     var _ {.codegenDecl: "const $2", exportc: astToStr(symbol).} = symbol
-    {.emit: "export { " & astToStr(symbol) & (if alias.len > 0: " as " & alias else: "") & " };" .}
+    {.emit: ["export { ", astToStr(symbol), (if name.len > 0: " as " & name else: ""), " };"] .}
+
+  # TODO: https://github.com/nim-lang/Nim/issues/16993
+  runnableExamples("-r:off"):
+    proc example = echo "This is exported as 'default' in JavaScript"
+    let example2 = "This is exported as 'another' in JavaScript"
+    const example3 = "This is exported as 'example3' in JavaScript"
+    jsExport(example, name = "default")  ## Alias for `export { symbol as default };`
+    jsExport(example2, name = "another") ## Alias for `export { symbol as another };`
```

Comment:
this fails if jsExport is called from a template
```nim
    template foo() =
      const x = "abc"
      jsExport(x)
    foo()
```
SyntaxError: Missing initializer in const declaration

that should be fixable in your PR